### DISCUSSION TITLE
How to get Ctrl + A working in Terminal Panel

### DISCUSSION THREAD

**Discussion by bobbykjack** [zed-industries/zed#17932] [Feature Requests]
I'm used to Ctrl + A taking me to the beginning of the line in a terminal, but in Zed's terminal panel, it just inserts a ^A. Even the suggested Cmd + A to move to beginning of the line does the same.

**Comment by JasperWallace** [comment]
In keymap.json add:

```
  {
    "context": "Terminal",
    "bindings": {
      "ctrl-a": ["terminal::SendKeystroke", "ctrl-a"]
    }
  }
```

**Reply by bobbykjack** [reply]
Thank you. For anyone else wondering, I think the issue was actually related to my shell, but either/both approach may help. In the end, I needed to add:

`set -o emacs`

to my .zshrc file.
